automatic/graphviz/legal/VERIFICATION.txt

```diff
@@ -1,21 +1,23 @@
-VERIFICATION
-
-Verification is intended to assist the Chocolatey moderators and community
-in verifying that this package's contents are trustworthy.
-
-Package can be verified like this:
-
-1. Go to
-
-   x32: https://graphviz.gitlab.io/_pages/Download/windows/graphviz-2.38.msi
```

Comment:
Version 2.38 is a very old version. The latest version is 2.44.1. If you indeed want to refer to version 2.38, the official location is https://www2.graphviz.org/Archive/stable/windows/graphviz-2.38.msi since a while back. I think we plan to remove https://graphviz.gitlab.io/_pages/Download/windows/graphviz-2.38.msi. /cc @mhansen

The are plenty of more references to 2.38 below.

Reply:
@magjac
The reference to version `2.38` is in a deleted section (GitHub pink) this is also the main reason for the PR.
Thanks for looking through the code.

Reply:
While that may be true here, there are other references to 2.38 added in this PR. E.g. automatic/graphviz/ver32.tmp .